src/bundle/Resources/public/scss/_field-group.scss

```diff
@@ -1,29 +1,44 @@
 @import 'custom';
 
 .ez-fieldgroup {
-	.ez-fieldgroup-name {
-		color: $ez-color-secondary;
-		border-bottom: 2px solid $ez-color-secondary;
-		padding-bottom: .5rem;
-	}
 
-	.ez-content-field {
-		padding-left: 2rem;
-		margin-bottom: 1.5rem;
+    h2 {
+        &.ez-fieldgroup-name {
+            color: $ez-color-secondary;
+            border-bottom: 2px solid $ez-color-secondary;
+            padding-bottom: .5rem;
+            font-size: 1.125rem;
+            font-weight: 400;
+        }
+    }
 
-		.ez-content-field-name {
-			background-color: $ez-ground-base;
-			margin-bottom: 0;
-			padding: .5rem 1.5rem;
-			border-top-right-radius: 5px;
-			border-top-left-radius: 5px;
-		}
+    h3 {
+        &.ez-fieldgroup-name {
```

Comment:
What do you think about making a generic selector: `.ez-fieldgroup-name` which contains color, border, padding-bottom and font-weight declarations? Then, in specific cases like: `h3.ez-fieldgroup-name` you'll apply the font-size rules only.

Reply:
I think you're right. It can be useful.
At some point we will have to rework `.ez-fieldgroup-name`
because it has to be specific to fieldgroup only (IMHO)
(cf. https://github.com/ezsystems/ezplatform-admin-ui/pull/113/files#diff-54b008e73d6d6a4eab6497a8dcb64a46R2)